Costume/100_Voltorb.yaml

```diff
@@ -0,0 +1,20 @@
+species: Voltorb
```

Comment:
Mareep is probably the better choice here because sheep.

Reply:
They also made Finneon the Woobat costume instead of, you know, Zubat, so I’m guessing they don’t want to take the easy route.

Reply:
Mareep was taken already with Skiddo. I wasn't sure what the rule was on reusing Pokemon for this set, so I went with a new Pokemon instead.

Reply:
Thought that was the concept, too. Hence why my costumes for Grookey and Scorbunny used Magby and Elekid instead of Mankey and Buneary.